client/packages/system/src/Stock/Components/Ledger/useLedgerColumns.ts

```diff
@@ -13,7 +13,7 @@ export enum ColumnKey {
   'DateTime' = 'datetime',
   'Time' = 'time',
   'Name' = 'name',
-  'Quantity' = 'quantity',
+  'UnitQuantity' = 'unitQuantity',
```

Comment:
I think the value still needs to be `quantity` - ledger is showing blank values now! 😁 

![Screenshot 2024-09-30 at 11 01 26 AM](https://github.com/user-attachments/assets/fdbb7b8a-0446-41f3-8b2a-73f800bd4499)